bitcoin/src/sign_message.rs

```diff
@@ -172,7 +179,11 @@ mod message_signing {
             pub fn from_base64(s: &str) -> Result<MessageSignature, MessageSignatureError> {
                 let bytes =
                     BASE64_STANDARD.decode(s).map_err(|_| MessageSignatureError::InvalidBase64)?;
```

Comment:
Not entirely related but it'd be nicer to check the length of `s` beforehand and then decode it into a fixed-sized array using `BASE64_STANDARD.decode_slice_unchecked(s, &mut byte_array)` so that we don't needlessly allocate.

Reply:
done